app/frontend/components/pages/login/loginPage.js

```diff
@@ -78,14 +63,10 @@ class LoginPage extends Component {
 
 module.exports = connect(
   (state) => ({
-    mnemonic: state.mnemonic,
```

Comment:
why those properties were removed from there. It isn't clear for me at first sight